README.md

```diff
@@ -44,6 +44,15 @@ Deletes `PersistentVolumeClaim` left behind by deleted `StatefulSet`, that are n
 - Waits for 7 days(configurable) before deleting
 - Ignores if `PersistentVolume` has `persistentVolumeReclaimPolicy` set to `Retain`
 
+## Remediator Policy
+You can define a remediator policy to control the following options:
+- `disabled_remediators`: All remediators are enabled by default unless listed in this option.
+    example:
+  ```json
+    {
+      "disabled_remediators": ["FailedPodRescheduler"]
+    }
+  ```
```

Comment:
would and env var be simpler ?

Reply:
much simpler, yes. Comma separated env?

Reply:
yes!

Reply:
looks like viper support env variable as well so enabled that instead (And updated readme)

```
❯ export DISABLED_REMEDIATORS=OldPodDeleter,CrashLoopBackOffRescheduler
❯ make dev
...
{"level":"info","message":"Skipping remediator as it is disabled.","remediator":"OldPodDeleter"}
{"level":"info","message":"Skipping remediator as it is disabled.","remediator":"CrashLoopBackOffRescheduler"}
....
```